magniv/build/build.py

```diff
@@ -105,9 +104,9 @@ def save_tasks(
         tasks_list = []
     if used_keys is None:
         used_keys = {}
+    if os.path.exists(f"{task_folder}/{reqs_pth}"):
```

Comment:
Won't this break the ability to have nested `requirements.txt`?

https://docs.magniv.io/documentation/file-structure

"Inside the /tasks/ folder, your project can be structured however you wish. Magniv will recursively search for tasks to orchestrate. In projects with nested subdirectories inside of /tasks/, Magniv will match each task with its closest requirements.txt (moving up each directory until reaching the /tasks)."

Reply:
no? just tested it and it worked fine for a nested dir with multiple reqs. it used to be in the os.walk for loop but doesn't use any vars from the os.walk

Reply:
Oooooh good call, you're right